src/sentry/static/sentry/app/components/list.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import theme from 'app/utils/theme';
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+
+export const List = styled('ul')`
+  list-style: none;
+  padding: 0;
+  margin-bottom: ${space(2)};
+
+  ol& {
+    counter-reset: numberedList;
+  }
+`;
+
+type ListItemProps = {
+  icon?: string | React.ReactNode;
+  children?: string | React.ReactNode;
+  className?: string;
+};
+
+const IconWrapper = styled('span')`
+  display: flex;
+  margin-right: ${space(1)};
+
+  /* Give the wrapper an explicit height so icons are line height with the
+   * (common) line height. */
+  height: 16px;
+  align-items: center;
+`;
+
+export const ListItem = styled(({icon, className, children}: ListItemProps) => (
+  <li className={className}>
+    {icon && (
+      <IconWrapper>
+        {typeof icon === 'string' ? <InlineSvg src={icon} /> : icon}
+      </IconWrapper>
+    )}
+    {children}
+  </li>
+))<ListItemProps>`
+  display: flex;
+  align-items: center;
+  position: relative;
+  padding-left: 34px;
+  margin-bottom: ${space(0.5)};
+
+  &:before,
+  & > span {
+    position: absolute;
+    left: 0;
+  }
+
+  ul & {
+    color: ${theme.gray700};
+    &:before {
+      content: '';
+      width: 6px;
+      height: 6px;
+      border-radius: 50%;
+      margin-right: ${space(2)};
+      border: 1px solid ${theme.gray700};
+      background-color: transparent;
+      left: 5px;
+      top: 10px;
+    }
+
+    ${p =>
+      p.icon &&
+      `
+      span {
```

Comment:
Do you want to target all spans or just the IconWrapper? The `children` could also contain spans and this selector would apply to them as well.

Reply:
Ah, very good point